### PR TITLE
Revert "Merge pull request #65 from mariuspot/master"

### PR DESCRIPTION
**Pull request by rjelierse**
The WhatsApp Sandbox has been properly integrated into Conversations API, so it shouldn't be accessed directly.

### CODE REVIEW

File: api/src/main/java/com/messagebird/MessageBirdClient.java [hunk 1]
```diff
@@ -950,7 +936,7 @@ public ConversationList listConversations() throws UnauthorizedException, Genera
      */
     public Conversation startConversation(ConversationStartRequest request)
             throws UnauthorizedException, GeneralException {
-        String url = String.format("%s%s/start", this.conversationsBaseUrl, CONVERSATION_PATH);
```
Comment (denizkilic) [inline api/src/main/java/com/messagebird/MessageBirdClient.java:953]: you should update also this.conversationsBaseUrl with CONVERSATIONS_BASE_URL on line 951

**Comment by denizkilic** [comment]
@olimpias Can you also check this PR?

**Comment by olimpias** [comment]
LGTM we only need to update `sendMessage` method as @denizkilic mentioned above. We added `sendMessage` method after #65 PR.

File: api/src/main/java/com/messagebird/MessageBirdClient.java (hunk 1 above)
Comment (rjelierse) [reply api/src/main/java/com/messagebird/MessageBirdClient.java:953]: Updated, thanks 👍

**Pull request merged**